content/guides/trash/index.md

```diff
@@ -0,0 +1,17 @@
+---
+rank: 50
+related_endpoints: []
+related_guides: []
+required_guides: []
+related_resources:
+  - folders
+  - files
+  - web-links
```

Comment:
`related_` and `required` items actually have no effect on index pages. Feel free to remove.

Reply:
And vice versa btw

Reply:
Let me know if I got it right this time. I stripped the index file down to just the rank since there didn't seem to be anything else to reference. Not sure about the other bits